### PR TITLE
Add piwik to SIC code list link-start page

### PR DESCRIPTION
**Pull request by aokunade**
BI-9803

Add piwik to SIC code list link-start page

### CODE REVIEW

File: src/views/advanced-search/index.html [hunk 1]
```diff
@@ -216,7 +216,7 @@ <h1 class="govuk-heading-l">Advanced company search</h1>
 
                       <input class='govuk-input govuk-!-width-full' id='sicCodes' name='sicCodes' type='text'>
 
-                      <p class='padded-top'> <a href='http://resources.companieshouse.gov.uk/sic/' target='_blank' class='govuk-link accordion-label'>Browse SIC codes (opens in new tab)</a></p>
+                      <p class='padded-top'> <a href='http://resources.companieshouse.gov.uk/sic/' target='_blank' class='govuk-link accordion-label' data-event-id = 'advanced-search-start-page-sic-link'>Browse SIC codes (opens in new tab)</a></p>
```
Comment (hwilliams1) [inline src/views/advanced-search/index.html:219]: There shouldn't be spaces in between the attribute, the equals sign and the value
e.g.
`data-event-id='advanced-search-start-page-sic-link'`

**Review by hwilliams1 — CHANGES_REQUESTED** [review]
Please remove the spaces between the attribute,equals sign and value

File: src/views/advanced-search/index.html [hunk 2]
```diff
@@ -216,7 +216,7 @@ <h1 class="govuk-heading-l">Advanced company search</h1>
 
                       <input class='govuk-input govuk-!-width-full' id='sicCodes' name='sicCodes' type='text'>
 
-                      <p class='padded-top'> <a href='http://resources.companieshouse.gov.uk/sic/' target='_blank' class='govuk-link accordion-label'>Browse SIC codes (opens in new tab)</a></p>
+                      <p class='padded-top'> <a href='http://resources.companieshouse.gov.uk/sic/' target='_blank' class='govuk-link accordion-label' data-event-id='advanced-search-start-page-sic-link'>Browse SIC codes (opens in new tab)</a></p>
```
Comment (caismith-ch) [inline src/views/advanced-search/index.html:219]: To fix the security issue raised in the PR, can `rel="noopener"` be added to the element please.

**Review by hwilliams1 — APPROVED** [review]
looks ok

**Pull request merged**